ocaml/runtime4/startup_byt.c

```diff
@@ -399,7 +399,7 @@ static void do_print_config(void)
          "false");
 #endif
   printf("windows_unicode: %s\n",
-#if WINDOWS_UNICODE
+#if defined(WINDOWS_UNICODE) && WINDOWS_UNICODE
```

Comment:
(The condition under `ocaml/runtime` is
slightly different, but -I think- equivalent.)

Reply:
Fixed